frontend/src/pages/CouponList/Coupon/index.tsx

```diff
@@ -0,0 +1,12 @@
+import { CouponWrapper } from './style';
+
+interface CouponProps {
+  frontImageUrl: string;
+  onClick: () => void;
+}
+
+const Coupon = ({ frontImageUrl, onClick }: CouponProps) => {
+  return <CouponWrapper src={frontImageUrl} onClick={onClick}></CouponWrapper>;
```

Comment:
단축구문 어떠신가요??
혹시 안에 추가로 무엇인가 들어갈 예정인가요??
```suggestion
  return <CouponWrapper src={frontImageUrl} onClick={onClick} />;
```